.env.test

```diff
@@ -0,0 +1,16 @@
+DEBUG=true
+SECRET_KEY='tests'
+DATABASE_URL='postgresql://postgres@localhost/opencraft'
```

Comment:
@bradenmacdonald Ok, I'll try that. Note that the current approach worked out of the box for me on the stock Ubuntus 14.04 that I tried, as well as CircleCI - have you changed the rights or password of your postgres user? The approach you give should also work there though I think, and it's more generic.